### DISCUSSION TITLE
How do I preview a STRING?

### DISCUSSION THREAD

**Discussion by ziontee113** [WASasquatch/was-node-suite-comfyui#150] [Q&A] [answered]
Hi, I'm very new to ComfyUI and especially custom nodes. I wonder if there's a way that I can "preview" a STRING output so I can easily try things out without having to save it in a text file. Thank you very much!

**Comment by ziontee113** [comment]
I just found the `Text to Console` node. But I wonder if there's a node that can preview text that looks like the native `Notes` node :thinking: ?

**Reply by WASasquatch** [reply, marked as answer]
Not through vanilla ComfyUI, and I am not having WAS-NS deploy custom nodes. You can voice your opinion over at ComfyUI and tell them that you too, want to be able to output text to nodes.

**Reply by ziontee113** [reply]
Thank you for the answer. I'll use `Text to Console` for now.
